front/app/components/UI/RemoveImageButton/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { Icon } from '@citizenlab/cl2-component-library';
+import React from 'react';
+import styled from 'styled-components';
+import { ScreenReaderOnly } from 'utils/a11y';
+import { useIntl } from 'utils/cl-intl';
+import { removeFocusAfterMouseClick } from 'utils/helperUtils';
+import messages from './messages';
+
+const RemoveIcon = styled(Icon)`
+  fill: #fff;
+  transition: all 100ms ease-out;
+`;
+
+const RemoveButton = styled.button`
+  width: 30px;
+  height: 30px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0;
+  margin: 0;
+  position: absolute;
+  top: 8px;
+  right: 8px;
```

Comment:
I slightly increased this to 12px. It was too close to the border imo (also for the existing ones).